app/controller.js

```diff
@@ -87,6 +88,21 @@ class Controller {
     }
   }
 
+  async addAddress(req, res) {
+    // Add an email address to the database.
+    try {
+      const { email } = req.body;
+      const exists = await Address.findOne({ where: { email } });
+      if (!email) return res.status(500).json({ error: 'No email provided.' });
+      if (exists) return res.status(418).json({ error: 'User already exists.' });
```

Comment:
I know you want to use this status code, but it's not necessarily the right one for the job here :)

Reply:
:smile: